<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitterWidget</class>
 <widget class="QWidget" name="SplitterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SplitterWidget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>561</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Text a decoder:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="BoutonDecoder">
          <property name="text">
           <string>Decoder</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Nom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Nom"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Pr√©nom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Prenom"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Age</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Age"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Poids</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Poids"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Taille</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Taille"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SplitterWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
